velox/docs/develop/scalar-functions.rst

```diff
@@ -336,9 +341,8 @@ Some of the defining features of these functions are:
 
 - take vectors as inputs and produce vectors as a result;
 - have access to vector encodings and metadata;
-- can be defined for generic input types;
-- allow for implementing lambda functions;
-- allow for pre-processing constant inputs and reusing the results across multiple batches, e.g. compile constant regular expressions once per query or thread of execution.
```

Comment:
Hi Masha, Thank you for improving the documentation! I wonder why would we like to remove this item, e.g., is it not true anymore?

Reply:
We enhanced simple functions to allow for pre-processing constant inputs. And reusing the results across multiple batches was always possible with simple functions. It was a mistake to mention it as vector function feature in the first place.